ci/cpu/cuml/build_cuml.sh

```diff
@@ -6,9 +6,9 @@ if [ "$BUILD_CUML" == '1' ]; then
   echo "Building cuML"
   CUDA_REL=${CUDA_VERSION%.*}
   if [[ -z "$PROJECT_FLASH" || "$PROJECT_FLASH" == "0" ]]; then
-    conda build conda/recipes/cuml --python=${PYTHON}
+    conda build -c rapidsai-nightly/label/testing conda/recipes/cuml --python=${PYTHON}
   else
-    conda build -c ci/artifacts/cuml/cpu/conda-bld/ --dirty --no-remove-work-dir conda/recipes/cuml --python=${PYTHON}
+    conda build -c rapidsai-nightly/label/testing -c ci/artifacts/cuml/cpu/conda-bld/ --dirty --no-remove-work-dir conda/recipes/cuml --python=${PYTHON}
```

Comment:
Curious: why is this change needed here?

Reply:
This is to pick the correct `libcumlprims` package from our channels. Going forward, whenever an MR in cumlprims is merged, it is added with `testing` label, and the targetted cuML PR in sync with that package must pass CI before we can move the cumlprims conda package to the main channel

Reply:
Got it. Thanks